plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -306,6 +307,32 @@ def get_realm_info_by_id(self, realm='master'):
             self.module.fail_json(msg='Could not obtain realm %s: %s' % (realm, str(e)),
                                   exception=traceback.format_exc())
 
+    def get_realm_keys_metadata_by_id(self, realm='master'):
+        """ Obtain realm public info by id
+
+        :param realm: realm id
+        :return: dict of real, representation or None if none matching exist
```

Comment:
Typo?
```suggestion
        :return: dict with realm configuration, representation or None if none matching exist
```

Reply:
Looks like a typo, right? :) tbh, I just copy pasted this from other descriptions and did not really get what `dict of real` could possibly mean. It makes sense now -- the typo propagated.
I'll have a proper look again what the returning data type is here

Reply:
Looks better. :-)